Submissions/Jockeys97.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Alessio Fantini",
+    "img": "https://avatars.githubusercontent.com/u/209752645?s=400&u=fa8dfa8dde13796c829e8863be2e1aa4fdb1474d&v=4",
+    "email": "alessiofant@gmail.com",
+    "links": {
+        "website": "https://jockeys97.github.io/Jockeys97-github.io/",
+        "linkedin": "https://www.linkedin.com/in/alessio-fantini-3a40a6220/",
+        "github": "https://github.com/Jockeys97"
+    },
+    "jobTitle": "Web Developer",
+    "location": "Italy"
```

Comment:
```
"location": {
    "city": "Example City",
    "state": "Example State",
    "country": "Example Country"
  }
 ```
 Please check the syntax, the curly braces, commas, etc. are important. Your entry hat errors in the `location` entry'.